proto/stream_plan.proto

```diff
@@ -482,6 +482,9 @@ enum StreamScanType {
 
   // ChainExecutor with upstream_only = true
   STREAM_SCAN_TYPE_UPSTREAM_ONLY = 4;
+
+  // ARRANGEMENT_BACKFILL corresponds to the `ArrangementBackfillExecutor`.
+  ARRANGEMENT_BACKFILL = 5;
```

Comment:
The prefix is only used as a namespace, and will be removed by the rust library (prost).

```suggestion
  STREAM_SCAN_TYPE_ARRANGEMENT_BACKFILL = 5;
```